rgw/admin/utils.go

```diff
@@ -12,6 +12,13 @@ const (
 )
 
 func buildQueryPath(endpoint, path, args string) string {
+	// Sometimes the API requires single URL key with no values like the Quota code
+	// So it passes the begining of the query in its path already, so we need to escape it
+	// and add a separator key instead
```

Comment:
This comment is not entirely clear to me. It's nothing major, I partly get the gist, but I suspect future readers my not entirely get it either. Perhaps we can come back another time and clarify it.

Reply:
I can add some examples.